File: examples-python/gpio.py
from gpioc.chips.K230 import *
import time
key = IO21
led = IO52
key.init(key.IN,key.PULL_UP)
led.init(led.OUT)


while True:
    time.sleep(0.1)
    if key.value() == 0:
        led.value(1)
    else:
        led.value(0)

File: examples-c/gpio.c
#include <stdio.h>
#include <time.h>
#include <unistd.h>
#include "../chips/h616.h"
#include "../src/gpioc.h"

int main()
{
    gpio_set_Mode(77, 1);

    while(1)
    {

        gpio_write(77, 0);
        sleep(1);
        gpio_write(77, 1);
        sleep(1);
    }
    return 0;
}